components/eamxx/tests/uncoupled/surface_coupling/surface_coupling_output.yaml

```diff
@@ -13,7 +13,7 @@ Field Names:
   - surf_lw_flux_up
   - surf_sens_flux
   - surf_evap
-  - surf_mom_flux
+  - wind_stress
```

Comment:
you renamed surf_mom_flux everywhere to wind_stress. it is a little confusing, since the old name was connected to surf_sens_flux and surf_evap (and all 3 are coming from the same codes). not sure whether i contributed to this confusion, last we talked about naming iirc it was only in context of QFLX limiter, not TMS.

Reply:
The variable name seemed wrong to me. Basically before this change, `surf_mom_flux_u` (same for `_v`) was imported from cpl variables `Faxx_taux`, which in `cam_cpl_indices.F90` in EAM is called "wind stress" with units `N/m2`. In EAM SHOC the input `uw_sfc` is referred to as "surface momentum flux" with units `m2/s2`, and is preprocessed in the interface as `uw_sfc = surf_mom_flux_u/rrho`. This seemed inconsistent, but I'm happy to change it back if the old naming made more sense.